lib/afterpay/payment.rb

```diff
@@ -21,12 +21,28 @@ def initialize(attributes = {})
       @open_to_capture_amount = Utils::Money.from_response(attributes[:openToCaptureAmount]) || Money.from_amount(0)
       @payment_state = attributes[:paymentState] || ""
       @merchant_reference = attributes[:merchantReference] || ""
-      @refunds = attributes[:refunds] || []
+      @refunds = !attributes[:refunds].nil? ? create_refund_objects_array(attributes[:refunds]) : []
```

Comment:
@sachinsaxena1996 wouldnt it be cleaner and more consistent to implement `from_response` class methods in `PaymentEvent` and `Refund` and use those here? Similar to the one in `Item`